app/scripts/colors/color-service.js

```diff
@@ -193,16 +194,20 @@
             var binSize = bins.length;
             var css = '';
 
+            // Every field except for year_build and floor_area needs a year prefix
+            var fieldWithYear = field === 'year_built' || field === 'floor_area' ? field :
```

Comment:
Might be clearer (and easier to update later) to make the list of fields to exclude from the postfix be a list in the Config, so it can be easily changed.